src/lib/router.js

```diff
@@ -4,6 +4,40 @@ const router = navaid();
 let isFirstRun = true;
 const domparser = new DOMParser();
 
+/**
+ * Provides a simple dynamic `import()` polyfill. Does not return the exports
+ * from the module: this is not how web.dev uses dynamic import.
+ *
+ * @param {string} src
+ * @return {!Promise<?>}
+ */
+window._import = (src) => {
```

Comment:
Done. Rollup only renders the dynamic function name at the very last possible step so it would clobber e.g. `function _import() ...` if it wasn't used, hence why it has to have global-side effects like this.